DQM/SiStripMonitorHardware/test/info.log

```diff
@@ -0,0 +1,11939 @@
+%MSG-i Geometry:  (NoModuleName) CSCGeometryESModule() 26-Jan-2018 15:07:28 CET  pre-events
```

Comment:
Hello @Jangbae is there any reason why this and other largish log files are included in this PR? It looks like a leftover of some test to me. Cheers and thanks.